Build/Scripts/PackageEditor.js

```diff
@@ -96,5 +97,68 @@ namespace('package', function() {
         });
 
     });
+    
+    task('linux_editor', {
+        async: true
+    }, function() {
+
+        // check if all the commands exist : fakeroot, dpkg-deb
+
+        var editorAppFolder = config.editorAppFolder;
+        var srcDir = config.artifactsRoot + "AtomicEditor/";
+        var dstDir = config.artifactsRoot + "Dist/";
+        var dstDeb = config.artifactsRoot + "AtomicGameEngine_0.0.8_amd64.deb";
+        
+        host.cleanCreateDir(dstDir);  // create new staging directory
+        fs.removeSync(dstDeb);  // remove old one, if there
+        
+        // copy in the two magic dirs
+        fs.copySync(config.atomicRoot + "Build/Linux/DEBIAN", dstDir + "DEBIAN");
+        fs.copySync(config.atomicRoot + "Build/Linux/usr", dstDir + "usr" );
+
+        // copy in the atomic dir
+        fs.copySync(editorAppFolder, dstDir + "usr/share/AtomicGameEngine" );
+       
+        //copy in menu pixmap
+        fs.copySync(config.atomicRoot + "Build/Linux/atomic_menu.xpm", dstDir + "usr/share/AtomicGameEngine/atomic_menu.xpm" );
+        
+        // get rid of some lintian errors
+        fs.removeSync( dstDir + "usr/share/AtomicGameEngine/Resources/ToolData/.gitignore");
+        fs.removeSync( dstDir + "usr/share/AtomicGameEngine/Resources/ToolData/CodeEditor/.gitignore");
+        fs.removeSync( dstDir + "usr/share/AtomicGameEngine/Resources/ToolData/Deployment/Android/assets/.gitignore");
+        fs.removeSync( dstDir + "usr/share/AtomicGameEngine/Resources/ToolData/ProjectTemplates/.gitignore");
+        fs.removeSync( dstDir + "usr/share/AtomicGameEngine/Resources/ToolData/TypeScriptSupport/.gitignore");
+
```

Comment:
I dont know if there is a globbing, this is part of making fewer lintian errors and warnings. Calling out specific files in another repo is fragile at best, I have a plan to write a program that will take the output of lintian and fix the files in the staging area, I don't know why one doesn't already exists, a new opportunity I guess.